Person.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+
+namespace GetDisplay
+
+{
+    class Person
+    {
+        public string Name { get; set; }
+        public int FavoriteNumber { get; set; }
+        public int NumberOfKids { get; set; }
+        public string FavoriteAnimal { get; set; }
+
+
+
+        public string AnswerQuestion(string question)
+        {
+            Console.WriteLine($"Hey {Name}, I have a question for you:");
+            Console.WriteLine(question);
+            return Console.ReadLine();
+        }
+
+        public int AnswerQuestionWithInteger(string question)
+        {
+            string stringAnswer = AnswerQuestion(question);
+            return int.Parse(stringAnswer);
+        }
+
+        public int FavoriteNumberAnswer(string question)
```

Comment:
When this runs, does it not turn a lot of the text to different colors besides just the numbers?  Also, this method has "side effects": it doesn't just get the user's favorite number, it also alters the UI